lib/vtex/client.rb

```diff
@@ -21,7 +21,8 @@ def connection
 
     def self.resources
       {
-        products: ProductResource
+        products: ProductResource,
+        categories: CategoryAndBrandResource
```

Comment:
I used the convention of resources in vtex catalog apis according to what we talked about but if you decide use just Categories it's Ok because brands is another resource